module/Finna/src/Finna/Db/Service/UserCardService.php

```diff
@@ -106,4 +107,30 @@ function ($card) use ($prefixes) {
         }
         return $cards;
     }
+
+    /**
+     * Get all users associated with a library card
+     *
+     * @param string $catUsername Catalog username
+     *
+     * @return array
+     */
+    public function getUsersForLibraryCard(string $catUsername): array
```

Comment:
Koska tämä ei palauta User-entiteettejä, niin ehkä jotain tällaista nimeksi, ettei hämää:
```suggestion
    public function getConnectedAccountInfoForLibraryCard(string $catUsername): array
```

Reply:
Metodi lisättävä myös UserCardServiceInterface:en.